app/locales/mr/intro.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private_kit": "प्राइवेट किट",
+    "intro1_para1": "सुरक्षा आणि गोपनीयता संरक्षणासह डिझाइन केलेले, एमआयटी प्रायव्हेट किट सुरक्षित लोकेशन लॉगिंगची पुढील पिढी आहे.",
+    "next": "पुढे",
+    "back": "मागे",
+    "start": "सुरु करा",
+    "intro2_title1": "100KB पेक्षा कमी",
+    "intro2_para1": "प्राइवेट किट चा ट्रेल जनरेटर आपल्या मोबाईल चा लोकेशन डेटा 100KB मध्ये स्टोअर करतो - एका photo पेक्षा कमी जागा.",
```

Comment:
मोबाईलचा लोकेशन-डेटा to make it easier to read
प्राइवेट किट चा ट्रेल जनरेटर आपल्या मोबाईलचा लोकेशन-डेटा फक्त 100KB मध्ये स्टोअर करतो, जी जागा एका photo पेक्षाही कमी आहे.